src/webview/apps/lightspeed/playbookGeneration/main.ts

```diff
@@ -40,46 +42,63 @@ window.addEventListener("load", () => {
   setListener("back-to-page2-button", backToPage2);
   setListener("open-editor-button", openEditor);
 
+  textArea = document.getElementById("playbook-text-area") as TextArea;
   setListenerOnTextArea();
-
   savedText = "";
-  savedOutline = "";
+
+  outline = new EditableList("outline-list");
+  outline.element.addEventListener("input", () => {
+    setButtonEnabled("reset-button", outline.isChanged());
+  });
+
+  // Detect whether a dark or a light color theme is used.
+  const element = document.getElementById("main-header");
+  if (element) {
+    const style = window.getComputedStyle(element);
+    const color = style.getPropertyValue("color");
+    const re = /rgb.?\((\d+), (\d+), (\d+)/;
+    const found = color.match(re);
+    if (found) {
+      const r = parseInt(found[1]);
+      const g = parseInt(found[2]);
+      const b = parseInt(found[3]);
+      darkMode = r > 128 && g > 128 && b > 128;
```

Comment:
does this work?
```js
window.matchMedia('(prefers-color-scheme: dark)').matches
```

Reply:
Let me try...

Reply:
> edit: eh nevermind -- this only checks the system setting and not the actual current VS Code theme.

I see. But I know this is not an elegant solution... If there is a better way, I would like to replace it.